README.md

```diff
@@ -1,4 +1,27 @@
-MyMedicare.gov BlueButton Data Pipeline
-=======================================
-
-Provides an Extract-Transform-Load (ETL) data pipeline, which moves data from CMS' Chronic Conditions data warehouse (CCW) into the CMS Blue  Button FHIR Server.
+# FHIR Backend Stress Test
+
+This is a test suite that utilizes jmeter, ansible and AWS to make a scalable
+set of load and performance tests for the FHIR backend test server.
+
+## Requirements
+
+This test suite has dependencies on the bluebutton-server application.  Clone
+that repo from: 
+
+https://github.com/HHSIDEAlab/bluebutton-server
+
+Then be sure to do a _mvn clean install_ in the bluebutton-server repository
+before attempting to build this test suite.
+
+## Running the tests
+
+These tests are currently using fixed servers and will require your username and
+password to execute.  Change to the _ansible_ subdirectory and use the following
+command line to execute the tests:
+
+    ansible-playbook fhir-stress-test.yml -u <username> -k
+
+This will execute the ansible test script and prompt for an ssh password to the
+servers whose configurations are protected in the ansible vault file found in
+group_vars.
+
```

Comment:
No need to change this, but for your own sanity, I strongly recommend using SSH keys for authentication instead of passwords. You should ensure they have a passphrase but use `ssh-agent` to cache that passphrase so you don't need to enter it more than once.

Reply:
Agreed, but they have some authentication setup on the test servers at the moment that still prompts for a login password even though I am using a valid ssh key.  This is not the passphrase but rather a login credential to the OS as it seems to be tied to my domain password.  I asked Curtis about it and he is looking into it.  I can add a blurb about ssh-agent for passphrases if you like?

Reply:
Oh, I ran into that, too, but you can resolve it by specifying on the client side that you want to use public key auth.